AmplifyPlugins/DataStore/Podfile

```diff
@@ -1,9 +1,9 @@
 platform :ios, '11.0'
 use_frameworks!
 
-AWS_SDK_VERSION = "2.12.2"
+AWS_SDK_VERSION = "2.12.6"
 
-pod 'SwiftFormat/CLI'
+pod 'SwiftFormat/CLI', '~> 0.42.0'
```

Comment:
Is there a reason not to just keep taking the latest? Are we seeing significant changes between versions?

Reply:
I still have this question.

Reply:
oh sorry I missed it.

I had some issues in the past with SwiftFormat `0.44` breaking when trying to format our code. I can try removing this and give it another shot.

That said, I think we need to add the `Podfile.lock` to the plugins as well, since I only recently realized that only core has the dependencies locked.